packages/lesswrong/components/users/UserMetaInfo.tsx

```diff
@@ -61,19 +61,19 @@ export const UserMetaInfo = ({user, hideAfKarma, hideWikiContribution, hideInfoO
   const infoClasses = classNames(infoClassName, classes.info);
 
   return <div className={classes.root}>
-      {karma && <div className={infoClasses}>
+      {(karma !== 0) && <div className={infoClasses}>
```

Comment:
@darkruby501 do we want to hide karma if it's 0?  If so, 👍 .

Reply:
I'm unsure. I assume if they have zero karma they'll also have 0 post and comments, so will be clean look. Doesn't seem critical as almost all users you'll look out will have karma